bot/client.js

```diff
@@ -57,4 +57,10 @@ client.on('message', (message) => {
   }
 });
 
+// This will send the welcome message for new users who join.
+client.on('guildMemberAdd', (member) => {
+  member.sendMessage(`${member.user.username}, Welcome to the Full Sail Armada! Enjoy your stay! and use !help to see list of commands!`);
```

Comment:
Consider that your implementation solves the problem, and elegantly, but falls just short in that your solution ​within the client.js file does not implement the controller you created. So in effect, any further maintenance will require duplicate efforts in keeping both messages identical.